### ISSUE TITLE
Use Carp 2012 to create list of items

### ISSUE THREAD

**Issue by Remi-Gau** [ohbm/eCobidas#34]
Without worrying too much about branching logic at the moment. 

With only one anatomical and one fMRI task.

- [x] cross-refs items from Carp 2012
- [x] itemize them as much as possible
- [x] note percentage of study reporting this item
- [x] create a simplified csv file
- [x] create scripts to convert neurovault metadata csv into a schema
- [x] create scripts to convert Carp 2012 csv into a schema
- [x] create one question for each item

**Comment by Remi-Gau** [comment]
I have added a python script from the tool folder of the schema-standardization repo.
It was used to convert csv files into a schema.

I think that trying to see how we can apply it to the csv file that describes the metadata from neurovault (added in the xlsx folder) and that is a subset of the COBIDAS could be a good starting point.

I have pushed all of those also on this branch for dev work on this.
https://github.com/Remi-Gau/COBIDAS_chckls/tree/neurovault2schema
